[UDC] [WEB] Build Fails

**Describe the bug**
<!--- A clear and concise description of what the bug is. -->

@rampallydheeraj  Please see this build failure.

- Command used

```bash
cd gimel/udc/udc-web
build/udc_ui
```

- Error Faced 
```bash
> ng build --env=dev --output-hashing all --sourcemaps false --base-href ./

Cannot read property 'config' of null
TypeError: Cannot read property 'config' of null
    at Class.run (/home/gimel/gimel/udc/udc-web/node_modules/@angular/cli/tasks/build.js:15:56)
    at Class.run (/home/gimel/gimel/udc/udc-web/node_modules/@angular/cli/commands/build.js:200:26)
    at resolve (/home/gimel/gimel/udc/udc-web/node_modules/@angular/cli/ember-cli/lib/models/command.js:273:20)
    at new Promise (<anonymous>)
    at Class.validateAndRun (/home/gimel/gimel/udc/udc-web/node_modules/@angular/cli/ember-cli/lib/models/command.js:251:12)
    at Promise.resolve.then.then (/home/gimel/gimel/udc/udc-web/node_modules/@angular/cli/ember-cli/lib/cli/cli.js:157:24)
    at tryCatch (/home/gimel/gimel/udc/udc-web/node_modules/rsvp/dist/rsvp.js:525:12)
    at invokeCallback (/home/gimel/gimel/udc/udc-web/node_modules/rsvp/dist/rsvp.js:538:13)
    at publish (/home/gimel/gimel/udc/udc-web/node_modules/rsvp/dist/rsvp.js:508:7)
    at flush (/home/gimel/gimel/udc/udc-web/node_modules/rsvp/dist/rsvp.js:2415:5)
    at _combinedTickCallback (internal/process/next_tick.js:132:7)
    at process._tickCallback (internal/process/next_tick.js:181:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

**To Reproduce**
<!--- Steps to reproduce the behavior -->

- See above listed steps

**Expected behavior**
<!--- A clear and concise description of what you expected to happen. -->

- Build Success

**Screenshots**
<!--- If applicable, add screenshots to help explain your problem. -->

**Potential fix**
<!--- If you are aware of a potential fix, please share the details.

--- a/.gitignore
+++ b/.gitignore
@@ -75,3 +75,16 @@ Thumbs.db
 
 # docker-files
 package-lock.json
+
+/target/
+.classpath
+.project
+.settings/
+logs/
+.DS_Store
+src/.DS_Store
+.idea/
+*.iml
+.vscode/
+udc/udc-web/dist/
+udc/udc-web/node_modules/

--- a/udc/udc-web/.angular-cli.json
+++ b/udc/udc-web/.angular-cli.json
@@ -1,0 +1,46 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "project": {
+    "name": "pcatalogweb"
+  },
+  "apps": [
+    {
+      "root": "src",
+      "outDir": "dist",
+      "assets": [
+        "assets"
+      ],
+      "index": "index.html",
+      "main": "main.ts",
+      "polyfills": "polyfills.ts",
+      "test": "test.ts",
+      "tsconfig": "tsconfig.app.json",
+      "testTsconfig": "tsconfig.spec.json",
+      "prefix": "app",
+      "styles": [
+        "styles.scss",
+        "theme.scss"
+      ],
+      "scripts": [],
+      "environmentSource": "environments/environment.ts",
+      "environments": {
+        "dev": "environments/environment.dev.ts"
+      }
+    }
+  ],
+  "lint": [
+    {
+      "project": "src/tsconfig.app.json"
+    },
+    {
+      "project": "src/tsconfig.spec.json"
+    },
+    {
+      "project": "e2e/tsconfig.e2e.json"
+    }
+  ],
+  "defaults": {
+    "styleExt": "scss",
+    "component": {}
+  }
+}

--- a/udc/udc-web/.editorconfig
+++ b/udc/udc-web/.editorconfig
@@ -1,0 +1,14 @@
+# Editor configuration, see http://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false

--- a/udc/udc-web/.eslintrc.json
+++ b/udc/udc-web/.eslintrc.json
@@ -1,0 +1,21 @@
+{
+  "parser": "typescript-eslint-parser",
+  "extends": ["airbnb", "plugin:jasmine/recommended"],
+  "plugins": [
+    "typescript",
+    "jasmine"
+  ],
+  "rules": {
+    "import/no-unresolved": "off",
+    "import/imports-first": "off",
+    "no-empty-function": "off",
+    "import/extensions": "off",
+    "import/prefer-default-export": "off",
+    "no-unused-vars": "off",
+    "no-undef": "off",
+    "comma-dangle": "off"
+  },
+  "env": {
+    "jasmine": true
+  }
+}

